code/__DEFINES/is_helpers.dm

```diff
@@ -91,19 +91,17 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 //More carbon mobs
 #define isalien(A) (istype(A, /mob/living/carbon/alien))
 
-#define isalienhumanoid(A) (istype(A, /mob/living/carbon/alien/humanoid))
-
 #define islarva(A) (istype(A, /mob/living/carbon/alien/larva))
 
-#define isalienadult(A) (istype(A, /mob/living/carbon/alien/humanoid) || istype(A, /mob/living/simple_animal/hostile/alien))
+#define isalienadult(A) (istype(A, /mob/living/carbon/alien/adult) || istype(A, /mob/living/simple_animal/hostile/alien))
```

Comment:
You cannot replace the first define with the second define, they are not equal and you will just be causing runtimes.

Reply:
why not?

Reply:
The old define I replaced was only used in 5 places:

Simple mob xenos are unable to generate this sound, no runtime.

https://github.com/tgstation/tgstation/blob/996b5a9c0df4e0a2fc4e8cb185e9c8863f660a0d/code/modules/mob/living/carbon/alien/emote.dm#L16-L30

Simple mob xenos are unable to swallow people, no runtime.

https://github.com/tgstation/tgstation/blob/45516f474147137d811d26ea72dd98a68953ad9f/code/modules/mob/living/carbon/alien/organs.dm#L316-L319

I tested spawning a bunch of slimes as a simple mob xeno, no runtime.

https://github.com/tgstation/tgstation/blob/70faa854b47fe784bb863e78a5b3a68f8464ab54/code/modules/mob/living/simple_animal/slime/life.dm#L345-L353

The queen early returns if trying to promote a simple mob that has no mind, so it doesn't runtime.

https://github.com/tgstation/tgstation/blob/bb4cce72e8d0d2121e8f8d084619435f2f50c920/code/modules/mob/living/carbon/alien/humanoid/queen.dm#L150-L159

Simple mob xenos have no runtimes for this either.

https://github.com/tgstation/tgstation/blob/43b00970496b9b4bce706b4a81503f769ffda8e5/code/game/machinery/doors/firedoor.dm#L122-L134

Reply:
i dont see how the two defines are not equivalent